solr/conf/schema.xml

```diff
@@ -169,8 +169,8 @@
   <copyField source="layer_slug_s"       dest="layer_slug_ti"       maxChars="100"/>
 
   <!-- core text search -->
-  <copyField source="*_ti"               dest="text" />
```

Comment:
Is it possible that this now would not use the text based analyzers that Solr might use on a text field?

Reply:
I think we're fine. The values of the string fields are copied into "text" which is a text_en field, so on index all the text_en *FilterFactory rules should apply. Here's what it looks like in Solr admin:

<img width="1279" alt="Screen Shot 2020-01-29 at 2 01 27 PM" src="https://user-images.githubusercontent.com/69827/73392787-1a336d80-42a0-11ea-893f-b9d2a003ffc1.png">

And indeed the 100 terms begin to show stemming, lowercase, etc.

<img width="1280" alt="Screen Shot 2020-01-29 at 2 04 23 PM" src="https://user-images.githubusercontent.com/69827/73392932-5cf54580-42a0-11ea-9284-ae2e020400e6.png">